documentation/blog/2024-10-25-run-x86-images-with-kubernetes-on-apple-silicon.md

```diff
@@ -0,0 +1,177 @@
+
+You have two options:
+
+1. Either start it by hand: `colima start --kubernetes --arch x86_64` or
+2. handroll your own LaunchDaemon:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>EnvironmentVariables</key>
+	<dict>
+		<key>PATH</key>
+		<string>/opt/homebrew/bin:/opt/homebrew/sbin:/usr/bin:/bin:/usr/sbin:/sbin</string>
+	</dict>
+	<key>KeepAlive</key>
+	<dict>
+		<key>SuccessfulExit</key>
+		<true/>
+	</dict>
+	<key>Label</key>
+	<string>de.weltraumschaf.colima</string>
+	<key>LimitLoadToSessionType</key>
+	<array>
+		<string>Aqua</string>
+		<string>Background</string>
+		<string>LoginWindow</string>
+		<string>StandardIO</string>
+	</array>
+	<key>ProgramArguments</key>
+	<array>
+		<string>/opt/homebrew/opt/colima/bin/colima</string>
+		<string>start</string>
+		<string>-f</string>
+		<string>--kubernetes</string>
+		<string>--arch</string>
+		<string>x86_64</string>
+	</array>
+	<key>RunAtLoad</key>
+	<true/>
+	<key>StandardErrorPath</key>
+	<string>/opt/homebrew/var/log/colima.log</string>
+	<key>StandardOutPath</key>
+	<string>/opt/homebrew/var/log/colima.log</string>
+	<key>WorkingDirectory</key>
+	<string>/Users/sst</string>
+</dict>
+</plist>
+```
+
+And store it in the file `~/Library/LaunchAgents/de.weltraumschaf.colima.plist`. Obviously, change "de.weltraumschaf" to whatever you like. Instead of Homebrew, now you need to use `launchctl` to interact with the LaunchAgent.
+
+## Install secureCodeBox with DefectDojo
+
+The rest is straight forward. To install _secureCodeBox_ simply execute (as documented [here](https://www.securecodebox.io/docs/getting-started/installation)):
+
+```shell
+helm --namespace securecodebox-system \ 
+  upgrade \ 
+  --install \
+  --create-namespace securecodebox-operator \
+  oci://ghcr.io/securecodebox/helm/operator
+```
+
+Then install the scanners you want, e.g. [Nmap](https://nmap.org/:
+```shell
+helm install nmap oci://ghcr.io/securecodebox/helm/nmap
+kubectl get scantypes
+```
+
+To install DefectDojo the easiest way is to clone their repo and install from it (as documented [here](https://www.securecodebox.io/docs/how-tos/persistence-storage/#defectdojo-kubernetes-setup)):
+
+```shell
+git clone https://github.com/DefectDojo/django-DefectDojo
+cd django-DefectDojo
+
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo update
+helm dependency update ./helm/defectdojo
+
+helm upgrade --install \
+  defectdojo \
+  ./helm/defectdojo \
+  --set django.ingress.enabled=true \
+  --set django.ingress.activateTLS=false \
+  --set createSecret=true \
+  --set createRabbitMqSecret=true \
+  --set createRedisSecret=true \
+  --set createMysqlSecret=true \
+  --set createPostgresqlSecret=true \
+  --set host="defectdojo.default.colima.local" \
+  --set "alternativeHosts={localhost}"
```

Comment:
the rabbitmq and mysql options aren't avaialble anymore so they don't have any effect anymore

```suggestion
helm upgrade --install \
  defectdojo \
  ./helm/defectdojo \
  --set django.ingress.enabled=true \
  --set django.ingress.activateTLS=false \
  --set createSecret=true \
  --set createRedisSecret=true \
  --set createPostgresqlSecret=true \
  --set host="defectdojo.default.colima.local" \
  --set "alternativeHosts={localhost}"
```

Reply:
Did you already removed this from documentation/docs/how-tos/persistence-storage.md? I think I've copied it from there.